src/pages/howly.jsx

```diff
@@ -41,13 +44,21 @@ export const pageQuery = graphql`
   query {
     takeOffImg: file(relativePath: { eq: "take-off.jpg" }) {
       childImageSharp {
-        fluid(maxHeight: 1000) {
+        fluid(maxHeight: 1000, quality: 50) {
           ...GatsbyImageSharpFluid
         }
       }
     }
+    howlyLogoImg: file(relativePath: { eq: "howly-logo.png" }) {
+      childImageSharp {
+        fluid(maxHeight: 1000) {
+          ...GatsbyImageSharpFluid_tracedSVG
```

Comment:
Blurred

Reply:
No worries, I'll change it, thought it looked better using traced though! As it outlined the word Howly.